### PR TITLE
Add a Safari Pinned Tab Icon

### PR DESCRIPTION
**Pull request by pat2man**
Icon adopted from openshift-logo.svg

Reference document: https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html

### CODE REVIEW

**Comment by sg00dwin** [comment]
@pat2man 
The mask-icon.svg looks fine. You just need to change the link path from images/openshift-logo.svg to images/mask-icon.svg

**Comment by pat2man** [comment]
@sg00dwin 😬 rebased.

File: app/images/mask-icon.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46 (44423) - http://www.bohemiancoding.com/sketch -->
+    <title>mask-icon</title>
+    <desc>Created with Sketch.</desc>
```
Comment (spadgett) [inline app/images/mask-icon.svg]: If it's not against the T&C for Sketch, I'd rather remove the description here and generator comment above.

Comment (pat2man) [inline app/images/mask-icon.svg]: @spadgett that makes sense but all the other SVGs also have the generator comment: https://github.com/openshift/origin-web-console/blob/master/app/images/logo-origin-thin.svg?short_path=fe5e643#L3

**Comment by pat2man** [comment]
@spadgett ran `grunt build` and rebased, passing.

File: app/images/mask-icon.svg (hunk 1 above)
Comment (spadgett) [inline app/images/mask-icon.svg]: Thanks for checking that. I'm not worried about it then

**Comment by spadgett** [comment]
Thanks @pat2man 

[merge]

**Pull request merged**